src/components/ImageGallery.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Card from './Card';
+import styles from '../../styles/ImageGallery.module.css';
+
+const ImageGallery = () => (
+  <section className={styles.gallery}>
+    <Card />
```

Comment:
Will there always be 9 of these? If so, preferably generate with a map based on an array (maybe just `[1, 2, 3, etc])`, to DRY things up the syntax front. You could then use the array member as the component `key`.

Reply:
Oh, no! There will only be one card eventually that then gets mapped through all the data from Redux. 
These are just placeholders for now. I needed to test that the scroll was visible and so I just copied and pasted it 9 times.